features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/CrashHandlerScenario.kt

```diff
@@ -22,9 +24,13 @@ internal class CrashHandlerScenario(
         val previousHandler = Thread.getDefaultUncaughtExceptionHandler()
 
         Thread.setDefaultUncaughtExceptionHandler({ t, e ->
-            Log.d("Bugsnag", "Intercepted uncaught exception")
+            Log.d("Bugsnag", "CrashHandlerScenario: Intercepted uncaught exception")
```

Comment:
Unfortunately the mazerunner fixture can only intercept messages routed through `config.logger`, so the E2E scenario won't detect this log message. You could create a property from `config.logger` and then log the message using that instead, which should resolve the issue.

Reply:
This isn't for logging to Maze-runner, just for debugging in general.  I just updated the message